src/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.TextBox.xaml

```diff
@@ -591,6 +600,13 @@
          TargetType="{x:Type TextBox}"
          BasedOn="{StaticResource MaterialDesignTextBoxBase}">
     <Setter Property="Padding" Value="{x:Static wpf:Constants.TextBoxDefaultPadding}" />
+    <Setter Property="wpf:BehaviorsAssist.Behaviors">
```

Comment:
Probably doesn't need to be addressed now, but I wonder if we should be concerned about someone using `BehaviorsAssist.Behaviors` to add in their own behaviors and inadvertently clearing this one. Might make for some difficult to diagnose issues later.

Reply:
That is a fair point! When thinking about it, it almost feels like the new behavior type I added should be public? That way you can at least add it back in. Would also make it easier when copy-pasting one of our styles to make certain tweaks...